### PR TITLE
Sp 432 display interventions list

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

File: app/action/index.njk [hunk 1]
```diff
@@ -3,6 +3,7 @@
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
```
Comment (Coupey) [inline app/action/index.njk]: Duplicated line

File: common/templates/components/accessible-autocomplete/template.njk [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<div class="govuk-form-group {% if error %}govuk-form-group--error{% endif %}">
+  {%if label %}
+    <label class="govuk-label" for={{ id }}>{{ label }}</label>
+  {% endif %}
+  {% if error %}
+    <span id="nationality-error" class="govuk-error-message">
```
Comment (Coupey) [inline common/templates/components/accessible-autocomplete/template.njk]: Change to a more generic id

File: app/action/interventionList/post.controller.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      .custom(value => {
+        return countWords(value) <= wordsAllowed
+      })
+      .withMessage(`Action description must be ${wordsAllowed} words or fewer`),
+    body('description')
+      .trim()
+      .escape(),
+    body('intervention')
+      .if(body('actionType').equals('intervention'))
+      .matches(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/, 'i')
```
Comment (Coupey) [inline app/action/interventionList/post.controller.js]: Should the front end care about what the intervention ID format is?

Comment (jonhindle) [reply app/action/interventionList/post.controller.js]: Arguable; the data type is a uuid, so how loose or heavy do you want to be with it? I've taken the  heavier perspective, that the post requires a UUID, so thats what I send.

Comment (Coupey) [reply app/action/interventionList/post.controller.js]: I think the argument is that we're only passing on what's been given to us, we're not generating it, using it directly in a way that depends on the format or mutating it - we're not ultimately responsible for the format of the UUID

Comment (jonhindle) [reply app/action/interventionList/post.controller.js]: Fair point. As is, the regexp means we don't have to sanitise too. Following discussion, leaving in for now and resolving.

**Pull request merged**